bamboo_twig_loader/src/TwigExtension/Render.php

```diff
@@ -42,13 +42,31 @@ public function getName() {
    *   The ID of the block to render.
    * @param array $params
    *   (optional) An array of parameters passed to the block.
+   * @param bool $wrapper
+   *   (Optional) Whether or not use block template for rendering.
    *
    * @return null|array
    *   A render array for the block or NULL if the block does not exist.
    */
-  public function renderBlock($block_id, array $params = []) {
+  public function renderBlock($block_id, array $params = [], $wrapper = FALSE) {
```

Comment:
the question about the default value is still open.

Reply:
I don't want to change the current working process, and so set it to false :)